runtime/lua/vim/lsp/callbacks.lua

```diff
@@ -54,9 +54,10 @@ M['textDocument/references'] = function(_, _, result)
   api.nvim_command("wincmd p")
 end
 
-local symbol_callback = function(_, _, result, _, bufnr)
```

Comment:
As far as I can tell `bufnr` isn't set. This was always `nil`.

Maybe the better solution would be to actually set it, because I think the solution I went for now has a race condition (user switches buffer before the callback is triggered)

Reply:
This should not be removed.
See this commit message.
https://github.com/neovim/neovim/pull/11931/commits/f157fdef7eccba6601c6d3b1475847466b60c25b

I think that if bufnr is nil and uri scheme is not file,  use uri as fname.